utils/staking-miner/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "staking-miner"
+name = "polkadot-staking-miner"
```

Comment:
I think `polkadot-staking-miner` is intended to be used for https://github.com/paritytech/staking-miner-v2. We could however use the name here and reserve `polkadot-staking-miner-v2` for that instead. What do you think? @niklasad1

Reply:
The intention is to deprecate/remove this crate, so I think you could mark it as `publish = false` for now
As this tool is tightly coupled to each polkadot release it doesn't make much sense to publish it on crates.io anyway.
 
//cc @kianenigma thoughts?

Reply:
> The intention is to deprecate/remove this crate, so I think you could mark it as `publish = false` for now

Ok. I've reverted the name to `staking-miner` for now.

Reply:
I think `staking-miner` currently can't be published anyhow because it depends on https://github.com/paritytech/substrate-debug-kit, which I couldn't find on crates.io